tasks/libs/package/size.py

```diff
@@ -104,15 +105,17 @@ def compute_package_size_metrics(
                 timestamp,
                 package_compressed_size,
                 tags=common_tags,
-            )
+                origin_metrics=get_origin_metrics(17, 0, 0),
```

Comment:
Can we create constants for these values? Something like `AGENT_ORIGIN_PRODUCT = 17` (not sure if it should be agent or agent-ci, or something else), same for the other ones. This will help in the future to understand where these values come from, what they mean, and help people that might need to use them one day

Reply:
I think we can link to internal documentation, as I did [here](https://github.com/DataDog/datadog-agent-buildimages/pull/652/files#diff-233838bf7d395bea1332104022ac1794462759e374eeefb25a04e662ecc7deecR3). But we don't want to "leak" any information about these values.

Reply:
That makes sense, but at the same time: 
* https://github.com/DataDog/datadog-agent/blob/4362f38fa5573a410c38d00fdf2525dab340632e/pkg/serializer/internal/metrics/iterable_series.go#L282-L295 
* https://github.com/DataDog/datadog-agent/blob/173a2ae8caf4a2787dbc9de98f1a720645173809/pkg/serializer/internal/metrics/origin_mapping.go

So up to you, I'm fine with the link to the doc (that could be broken one day)

Reply:
Yeah, linking to doc over "documenting" the values was suggested from the origins folks. I think my thought was more towards the naming of the constants in which, A) the name isn't _really_ accurate, and B) could be considered leaking.

What about just `ORIGIN_PRODUCT|CATEGORY|ETC` so we can still reuse the values, but it's more obfuscated?

Reply:
> What about just ORIGIN_PRODUCT|CATEGORY|ETC so we can still reuse the values, but it's more obfuscated?

Yes that's a good middle ground, this + the link to the doc and that's perfect!